include/mbedtls/x509_crt.h

```diff
@@ -70,6 +70,7 @@ typedef struct mbedtls_x509_crt
     mbedtls_x509_time valid_from;       /**< Start time of certificate validity. */
     mbedtls_x509_time valid_to;         /**< End time of certificate validity. */
 
+    mbedtls_x509_buf pk_raw;
```

Comment:
Note: this changes the ABI. Not a problem, but needs to be tracked.

Reply:
Actually, we only need this for `!MBEDTLS_SSL_KEEP_PEER_CERTIFICATE`, so we could guard this change accordingly to not change the ABI in the default configuration. WDYT?

Reply:
Don't we already have an ABI in libmbedx509 change from one of the preceding PRs? If we do, then I'm inclined to no care about this one. Otherwise, yes, let's use guards so that it only happens in non-default configs.

Btw, the guards might still be useful to save some RAM in the default config, though.

Reply:
Yes, we already have an ABI change in another X.509 PR, so considering that, a further change does no harm. However, I think it's still cleaner to not add this field if we don't need it, so I'll remove it.

Reply:
@mpg Sorry, I have to withdraw that; the X.509 module shouldn't depend on a SSL configuration. I'll leave the field unconditional.

Reply:
Excellent point, thanks for noticing.